common/src/main/java/com/leo/utils/LoadConfig.java

```diff
@@ -19,7 +19,7 @@ public static LoadConfig getInstance() {
     if (instance == null) {
       synchronized (LoadConfig.class) {
         if (instance == null) {
-          instance = new LoadConfig("src/main/resources/config.properties");
+          instance = new LoadConfig("server/src/main/resources/config.properties");
```

Comment:
i think it should be ./src/main since loadconfig will be used by both client and server, if you do it like this client will not be able to read properties file